cli/ballerina-cli/src/main/resources/create_cmd_templates/main/main.bal

```diff
@@ -1,5 +1,7 @@
 import ballerina/io;
 
+# Prints `Hello World`.
+
 public function main() {
-    io:println("Hello World!");
+    io:println(Hello, World!);
```

Comment:
missing double quotes in print message

Reply:
We don't need to maintain a separate file for the `-t main`. We can use one `main.bal` for both `default` and `main` templates.